core/src/main/java/org/apache/calcite/rel/core/JoinRelType.java

```diff
@@ -152,8 +152,12 @@ public JoinRelType cancelNullsOnRight() {
     }
   }
 
-  /** Transform this JoinRelType to CorrelateJoinType. **/
-  public CorrelateJoinType toLinq4jCorrelateJoinType() {
+  /**
+   * Transforms this JoinRelType to CorrelateJoinType.
+   * @deprecated Use {@link #toLinq4jJoinType()}
```

Comment:
If we remove the `#toLinq4jJoinType()` (see comment below) then here instead of giving a link to that method we could alternatively propose using a switch statement (or point to `EnumUtils`).

Reply:
Ok for me to point to EnumUtils.